### DISCUSSION TITLE
Netflix

### DISCUSSION THREAD

**Discussion by dont274** [AndreyPavlenko/Fermata#169] [General]
Can you add netflix app too.

**Comment by marcok8787** [comment]
Quote

**Comment by AndreyPavlenko** [comment]
I can't add any app. I could only add support for some service, but, afaik, Netflix doesn't provide any public API.

**Comment by fcaronte** [comment]
I see Google Chromecast features, do you this is possible to enable reverse mode to use Netflix app to cast video to "fermata Auto"? Just an idea...

**Comment by AndreyPavlenko** [comment]
This new feature is a sender, but not receiver. I.e. Fermata can send media to receivers, but not vice versa.

**Reply by fcaronte** [reply]
Yes i see, but technically can be possible to do the Opposite? For firetv (android) there are some app doing that

**Reply by AndreyPavlenko** [reply]
Probably, it's possible. The receiver implementation is required.

**Reply by AndreyPavlenko** [reply]
Do you know any cast receiver app for Android?

**Reply by fcaronte** [reply]
I don't know different from free and pro version, but I use pro version of this one

https://play.google.com/store/apps/details?id=com.softmedia.receiver.lite

**Reply by fcaronte** [reply]
But I don't try if can be used to receive netflix

**Reply by AndreyPavlenko** [reply]
Thanks. Seems, it does not work with unregistered apps - https://developers.google.com/cast/docs/registration.

**Reply by fcaronte** [reply]
Oh thanks so no way 😬
